config/locales/custom/admin.yml

```diff
@@ -196,3 +196,10 @@ ro:
         create: Creează o nouă înregistrare de date locale
       new:
         creating: Creează o nouă înregistrare de date locale
+    shared:
+      color_help: Format hexadecimal
+
+en:
+  admin:
+    shared:
+      color_help: Hexadecimal format
```

Comment:
```suggestion
```

These texts are already in CONSUL, so maybe we can remove these lines :thinking:.